src/components/media/controls/commonControls.tsx

```diff
@@ -1,31 +1,26 @@
-import './controls.css'
+import './commonControls.css'
 
 import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown'
 import KeyboardArrowUpIcon from '@mui/icons-material/KeyboardArrowUp'
 import Box from '@mui/material/Box'
 import Button from '@mui/material/Button'
-import Icon from '@mui/material/Icon'
-import IconButton from '@mui/material/IconButton'
 import LinearProgress from '@mui/material/LinearProgress'
 import Slider from '@mui/material/Slider'
 import Typography from '@mui/material/Typography'
 import React, { useState } from 'react'
 
 import { formatDurationTime } from '../../helper'
+import { MediaIconButton } from './mediaIconButton'
 
-interface MediaIconButtonProps {
-  onClick: () => void
-  action: 'play' | 'pause' | 'forward' | 'replay' | 'volumeUp' | 'volumeOff'
-  className?: string
-}
-
-interface MediaControls {
+export interface MediaControls {
   mediaElement: HTMLMediaElement
+  color?: string
 }
 
-interface TranscriptToggleProps {
-  isTranscriptShown: boolean
-  setIsTranscriptShown: () => void
+interface Toggle {
+  active: boolean
```

Comment:
```suggestion
  isActive: boolean
```

I do think that this interface is too generic and does not explain anything.

Reply:
What about:
```
interface ToggleButton {
  isSomeState: boolean
  setIsSomeState: () => void
}
```